app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -33,6 +33,12 @@ def perform(_now)
     )
 
     started_at = Time.zone.now
+
+    enrollments.each do |enroll|
+      enroll.update(last_batch_claim_at: started_at)
+    end
```

Comment:
if `enrollments` is a scope, you can call `.update_all` on them instead of doing one-by-one?
```suggestion
    enrollments.update_all(last_batch_claim_at: started_at)
```